exporter/sapmexporter/README.md

```diff
@@ -50,6 +52,7 @@ exporters:
     endpoint: https://ingest.YOUR_SIGNALFX_REALM.signalfx.com/v2/trace
     max_connections: 100
     num_workers: 8
+    log_detailed_response_from_trace_ingest: true
```

Comment:
Even if we want this option, let's make it shorter `log_detailed_response`? I seems too verbose

Reply:
Updated